internal/exec/selected/selected.go

```diff
@@ -153,21 +162,14 @@ func applySelectionSet(r *Request, s *resolvable.Schema, e *resolvable.Object, s
 					}
 				}
 
-				packedDirectives, err := packDirectives(fe, r.Vars)
-				if err != nil {
-					r.AddError(errors.Errorf("%s", err))
-					return
-				}
-
 				fieldSels := applyField(r, s, fe.ValueExec, field.SelectionSet)
 				flattenedSels = append(flattenedSels, &SchemaField{
-					Field:            *fe,
-					Alias:            field.Alias.Name,
-					Args:             args,
-					PackedArgs:       packedArgs,
-					PackedDirectives: packedDirectives,
-					Sels:             fieldSels,
-					Async:            fe.HasContext || fe.ArgsPacker != nil || fe.HasError || HasAsyncSel(fieldSels),
+					Field:      *fe,
+					Alias:      field.Alias.Name,
+					Args:       args,
+					PackedArgs: packedArgs,
+					Sels:       fieldSels,
+					Async:      fe.HasContext || fe.ArgsPacker != nil || fe.HasError || HasAsyncSel(fieldSels),
```

Comment:
Should fields with directive packers be async too? I guess yes... 
This operation takes a bit longer than a normal field resolution. 🤔 
I think now is the time to add one more condition:
```suggestion
					Async:      fe.HasContext || fe.ArgsPacker != nil || fe.DirectiveVisitors != nil || fe.HasError || HasAsyncSel(fieldSels),
```